<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KMainWindowClass</class>
 <widget class="QMainWindow" name="KMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>531</width>
      <height>371</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>输入半径：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="radiusEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>结果是：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="areaButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>求面积</string>
    </property>
   </widget>
   <widget class="QPushButton" name="circumferenceButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>求周长</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>989</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="kmainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
